pkg/validation/strfmt/format.go

```diff
@@ -231,3 +233,40 @@ func (f *defaultFormats) Parse(name, data string) (interface{}, error) {
 	}
 	return nil, errors.InvalidTypeName(name)
 }
+
+// scan provides a generic implementation of sql.Scanner interface's Scan function for basic string formats.
+func scan[T ~string](r *T, raw interface{}) error {
+	switch v := raw.(type) {
+	case []byte:
+		*r = T(v)
+	case string:
+		*r = T(v)
+	default:
+		return fmt.Errorf("cannot sql.Scan() strfmt.StringFormat from: %#v", v)
```

Comment:
🤨  are the sql.Scan errors load-bearing or can we stop outputting that?

Reply:
For some reason most of the string formats in kube-openapi support `sql.Scanner` and output errors like this.  But a few don't.  I'll just not support it and open an issue to drop it from the other string formats.

Reply:
https://github.com/kubernetes/kube-openapi/issues/536